templates/typescript_gapic/test/gapic-$service-$version.ts.njk

```diff
@@ -8,11 +8,16 @@ const {PassThrough} = require('stream');
 {% endif %}
 
 const FAKE_STATUS_CODE = 1;
-const error = new Error();
-error.code = FAKE_STATUS_CODE;
+class fakeError{
+    code: number;
+    constructor(n: number){
+        this.code = n;
+    }
+}
+const error = new fakeError(FAKE_STATUS_CODE);
 
-function mockSimpleGrpcMethod(expectedRequest, response, error) {
-    return function(actualRequest, options, callback) {
+function mockSimpleGrpcMethod(expectedRequest: any, response: any, error: any) {
```

Comment:
Using `any` is forbidden by our rules

Reply:
should we give the real request type here? Thought it's mocking method and it can take `any` type as argument type.

Reply:
Maybe just `{}` since we know those are objects. `any` is a forbidden keyword that means "I don't want any type checks", and `gts` will trigger on it.

Reply:
got it, thanks!